patches/nginx-1.17.4-gcc-maybe-uninitialized-warning.patch

```diff
@@ -7,5 +7,5 @@
 -    ngx_http_core_srv_conf_t  *cscf;
 +    ngx_http_core_srv_conf_t  *cscf = NULL;
  
-     hc = r->http_connection;
- 
+ #if (NGX_SUPPRESS_WARN)
+     cscf = NULL;
```

Comment:
Is it because of new changes in the nginx core?

Reply:
This was changed since 2014, please check nginx/nginx@8d97a2e. In my opition, this patch should be EOL(the mention was also committed as the message body).